quiz.js

```diff
@@ -0,0 +1,49 @@
+var height = document.getElementById('height');
+var character = document.getElementById('character');
+var button = document.getElementById('button');
+var branch = [];
+
+button.addEventListener('click', validate);
+height.addEventListener('keypress', keypressHandler);
+character.addEventListener('keypress', keypressHandler);
+
+function keypressHandler(event) {
+	if (event.keyCode === 13) {
+		validate();
+	}
+}
+
+function validate() {
```

Comment:
@daveharmswebdev great start on this. Since you're one of the first to submit, and you seem to have a nice grasp of the basics of JavaScript I'm going to challenge you to take it to the next level.  Let's talk Single Responsibility Principle.

The main concept in the Principle is that each function should do one thing only, with no side-effects.  In the code you've written, the `validate()` function violates the principle.  It validates the input fields, but it also invokes the `makeTree()` function.

Take a stab at rewriting your functions to uphold the SRP.